test/fuzz/fuzz_runner.cc

```diff
@@ -24,6 +25,7 @@ PerTestEnvironment::~PerTestEnvironment() { TestEnvironment::removePath(test_tmp
 
 void Runner::setupEnvironment(int argc, char** argv, spdlog::level::level_enum default_log_level) {
   Event::Libevent::Global::initialize();
+  Http::Http2::initializeNghttp2Logging();
```

Comment:
I'm confused, doesn't this actually enable nghttp2 tracing? Is the PR title correct?

Reply:
As a result of the build setting of `DEBUGBUILD` in the previous PR, nghttp2 will start tracing regardless of what we do Envoy-side. We need to register a callback to intercept and shovel it through Envoy's loggers rather than the default behavior which is to send to stderr.

Reply:
Ah OK I see.